apps/api/src/database/seed/data-static.js

```diff
@@ -296,6 +302,15 @@ export const subSectors = [
 			displayNameCy: 'Transfer of Water Resources'
 		},
 		sectorName: 'water'
+	},
+	{
+		subSector: {
+			name: 'training',
+			abbreviation: 'TRAIN01',
+			displayNameEn: 'Training',
+			displayNameCy: 'Training'
+		},
+		sectorName: 'training'
 	}
 ];
 
```

Comment:
Is it worth also adding a small DB migration script to insert these new sector / subsector types, rather than having to run db:seed?